### PR TITLE
CRUD-project

### PR DESCRIPTION
**Pull request by daothanhcam**

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Project < ActiveRecord::Base
+  belongs_to :team
+
+  def team_name
+    Team.find_by_id(team_id).name
```
Comment (dieunb) [inline app/models/project.rb]: try `delegate`

File: app/views/admin/teams/_team.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <tr>
   <td><%= link_to team.name, admin_team_path(team) %></td>
   <td>|<%= link_to " ", edit_admin_team_path(team), class: "glyphicon glyphicon-edit" %></td>
-  <td>|<%= link_to " ", admin_team_path(team), 
+  <td>|<%= link_to " ", admin_team_path(team),
   method: :delete, data: {confirm: t("notice.yousure")}, class: "glyphicon glyphicon-trash" %></td>
```
Comment (dieunb) [inline app/views/admin/teams/_team.html.erb]: indent

File: app/views/admin/teams/_team.html.erb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <tr>
   <td><%= link_to team.name, admin_team_path(team) %></td>
   <td>|<%= link_to " ", edit_admin_team_path(team), class: "glyphicon glyphicon-edit" %></td>
-  <td>|<%= link_to " ", admin_team_path(team), 
-  method: :delete, data: {confirm: t("notice.yousure")}, class: "glyphicon glyphicon-trash" %></td>
-</tr>
\ No newline at end of file
+  <td>|<%= link_to " ", admin_team_path(team),
+  method: :delete, data: {confirm: t("notice.yousure")},
```
Comment (phonghenry14) [inline app/views/admin/teams/_team.html.erb]: indent

File: app/models/team.rb [hunk 4]
```diff
@@ -13,4 +13,8 @@ class Team < ActiveRecord::Base
   after_create :log_create
   after_update :log_update
   after_destroy :log_destroy
+
+  def team_name
```
Comment (dieunb) [inline app/models/team.rb]: no need

File: app/models/project.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class Project < ActiveRecord::Base
+  belongs_to :team
+
+  delegate :team_name, to: :team
```
Comment (dieunb) [inline app/models/project.rb]: only `:name` and add `prefix: true`, you can call `team_name`

Comment (daothanhcam) [inline app/models/project.rb]: cái này hay quá anh ạ :)

File: app/models/project.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+class Project < ActiveRecord::Base
+  belongs_to :team
+
+  delegate :team_name, to: :team, prefix: true
```
Comment (dieunb) [inline app/models/project.rb]: `name` not `:team_name`

**Pull request merged**